src/lib.rs

```diff
@@ -550,7 +550,7 @@ where
 }
 
 /// The error type returned by methods in this crate.
-#[derive(Debug)]
+#[derive(Debug, PartialEq)]
```

Comment:
this is needed for some applications which has its own Error and it wants to derive PartialEq